bitcoinaudit/TweetCreator.py

```diff
@@ -2,35 +2,33 @@
 from decimal import Decimal
 
 class TweetCreator(object):
-	def __init__(self, height_current, total_current, height_yesterday, total_yesterday):
-		self.height_current = height_current
-		self.height_yesterday = height_yesterday
-		self.total_current = total_current
-		self.total_yesterday = total_yesterday
-
-	def block_height_increase_since_yesterday(self):
-		return self.height_current - self.height_yesterday
-
-	def total_increase_since_yesterday(self):
-		return self.total_current - self.total_yesterday
-
-	def create_tweet(self):
-		total_formatted = f"{self.total_current:,}"
-		increase_blocks_formatted = f"{self.block_height_increase_since_yesterday():,}"
-		increase_total_formatted = f"{self.total_increase_since_yesterday():,}"
-		increase_total_formatted = increase_total_formatted.rstrip('0').rstrip('.')
-
-		return self.msg_template().format(
-			height=self.height_current,
-			total=total_formatted,
-			increase_blocks=increase_blocks_formatted,
-			increase_total=increase_total_formatted)
-
-	def msg_template(self):
-		return """#Bitcoin block {height}
+    def __init__(self, height_current, total_current, height_yesterday, total_yesterday):
+        self.height_current = height_current
+        self.height_yesterday = height_yesterday
+        self.total_current = total_current
+        self.total_yesterday = total_yesterday
+
+    def block_height_increase_since_yesterday(self):
+        return self.height_current - self.height_yesterday
+
+    def total_increase_since_yesterday(self):
+        return self.total_current - self.total_yesterday
+
+    def create_tweet(self):
+        return self.msg_template().format(
+            height=self.height_current,
+            total=f'{self.total_current:,}',
+            increase_blocks=f'{self.block_height_increase_since_yesterday():,}',
+            increase_total=f'{self.total_increase_since_yesterday():,.2f}'
+        )
+
+    def msg_template(self):
+        return """\
+#Bitcoin block {height}
 
 Total supply: {total} BTC
 
 Increase since yesterday:
 +{increase_blocks} blocks
-+{increase_total} BTC"""
\ No newline at end of file
++{increase_total} BTC
```

Comment:
This would cause a new line in the tweet

Reply:
Really? According to the spec the first and last empty line should actually be removed. But if you want to be safe, append a backslash to the second to last line. The closing triple quotes should always stand alone on a line.

Reply:
I've tried it with the """ on the same line before and the tweet was rendered with a blank line at the end.
But I'm not 100% sure :-p